os-eap-launch/added/launch/resource-adapters-common.sh

```diff
@@ -106,7 +107,13 @@ function inject_resource_adapters_common() {
       resource_adapter="${resource_adapter}<transaction-support>$transaction_support</transaction-support>"
     fi
 
-    resource_adapter="${resource_adapter}<connection-definitions><connection-definition class-name=\"${ra_class}\" jndi-name=\"${ra_jndi}\" enabled=\"true\" use-java-context=\"true\">"
+    resource_adapter="${resource_adapter}<connection-definitions><connection-definition"
+
+    if [ -n "${RA_TRACKING}" ]; then
```

Comment:
in this case this is resource-adapter specific, so it should use the ra_prefix_TRACKING